streaming/index.js

```diff
@@ -294,6 +276,7 @@ const startServer = async () => {
 
     callbacks.forEach(callback => callback(json));
   };
+  redisSubscribeClient.on("message", onRedisMessage);
```

Comment:
Thank you again @ThisIsMissEm for the reviews. I brought this up and out of the actual subscribe loop, but I'm not sure if I want to bring this up higher - I'd have to move the entire assignment of `onRedisMessage` further up so I opted to instead place the `.on()` here.